modules/gapi/include/opencv2/gapi/python/python.hpp

```diff
@@ -0,0 +1,55 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2021 Intel Corporation
+
+
+#ifndef OPENCV_GAPI_PYTHON_API_HPP
+#define OPENCV_GAPI_PYTHON_API_HPP
+
+#include <opencv2/gapi/gkernel.hpp>     // GKernelPackage
+#include <opencv2/gapi/own/exports.hpp> // GAPI_EXPORTS
+
+namespace cv {
+namespace gapi {
+namespace python {
+
+GAPI_EXPORTS cv::gapi::GBackend backend();
+
+using Impl = std::function<cv::GRunArgs(const cv::GArgs&,
+                                        const cv::GTypesInfo&,
+                                        const cv::GTypesInfo&)>;
```

Comment:
Is it required here or can it be part of `GPythonKernel`?

Reply:
I thought about it, but it's used in `GPythonFunctor` as well, so let it be, if it isn't a big deal